akka-js-actor-stream/js/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -0,0 +1,875 @@
+/*
```

Comment:
Can you please mark with a comment where this file has been changed from upstream?

Reply:
Thanks for the fast reply! :) I've the comments in the same style as I found it in other files with `// SCALA.JS ...`